pkgs/by-name/to/tone/package.nix

```diff
@@ -22,12 +29,18 @@ buildDotnetModule rec {
   dotnet-sdk = dotnetCorePackages.sdk_6_0;
```

Comment:
.NET 6 is EOL (https://github.com/NixOS/nixpkgs/issues/326335), can we patch this package to use .NET 8 or 9? Best would be to update it upstream

Reply:
Yeah, I tried updating it to 8 but that definitely doesn't work. I'm not sure what I'd need to patch exactly, as I'm not familiar with C#, but I can open an issue upstream

Reply:
Opened https://github.com/sandreas/tone/issues/77

Reply:
Yes, please open an issue upstream, thanks!

You can update to .NET 8 by making these changes:
```diff
-  dotnet-sdk = dotnetCorePackages.sdk_6_0;
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnetFlags = [ "-p:TargetFramework=net8.0" ];
```

Then regenerating the `deps.nix` file with this command: `$(nix-build . -A tone.fetch-deps --no-out-link)`

Reply:
Until upstream doesn't officially supports .NET 8/9, I think we should use the above override

Reply:
That's not so bad, thanks! That works great!